src/test/java/org/fundacionjala/coding/omar/movies/CustomerTest.java

```diff
@@ -0,0 +1,163 @@
+package org.fundacionjala.coding.omar.movies;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Powered by Omar Limbert Huanca Sanchez - AT06.
+ * This class contains UnitTests for Movies App.
+ */
+public class CustomerTest {
+
+    private static final int CAPACITY = 100;
+    private static final String BLACK_PANTHER = "Black Panther";
+    private Customer customer;
+
+    /**
+     * This method is for test overViewNoRentedMovies().
+     */
+    @Before
+    public void customerTestInit() {
+        customer = new Customer("Pepito Perez");
+
+    }
+
+    /**
+     * This method is for test overViewNoRentedMovies().
+     */
+    @Test
+    public void testGetRentalOverviewNoRentedMovies() {
+        StringBuilder expectedOverviewBuilder = new StringBuilder(CAPACITY);
+
+        expectedOverviewBuilder.append("Customer: Pepito Perez"
+                + "\n====================================="
+                + "\nTotal 0.0\nYou earned 0 frequent renter point(s)");
+
+        assertEquals(expectedOverviewBuilder.toString(), customer.getRentalOverview());
+    }
+
+    /**
+     * This method is for test to add Children movie for 2 days.
+     */
+    @Test
+    public void testAddRentalChildrenChildrenForTwoDays() {
+
+        final int number2 = 2;
+        final int number1 = 1;
+        final double number1point5 = 1.5;
+
+        MovieChildren childrensMovie = new MovieChildren("The Minions");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "The Minions" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)